This patch is a workaround to disable the use of sgen.exe after the build,
since sgen.exe is not available in Mono. The trick works because we use a
syntax which is common to sh and cmd.exe, but we also use `true` which is part
of UNIX coreutils.
true || windows-part

--- a/KeePass/KeePass.csproj
+++ b/KeePass/KeePass.csproj
@@ -1249,6 +1249,6 @@
   </Target>
   -->
   <PropertyGroup>
-    <PostBuildEvent>"$(FrameworkSDKDir)bin\sgen.exe" /assembly:"$(TargetPath)" /force /nologo /compiler:/keycontainer:VS_KEY_33430356D8D7D1B8 /compiler:/delaysign-</PostBuildEvent>
+    <PostBuildEvent>true || "$(FrameworkSDKDir)bin\sgen.exe" /assembly:"$(TargetPath)" /force /nologo /compiler:/keycontainer:VS_KEY_33430356D8D7D1B8 /compiler:/delaysign-</PostBuildEvent>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+</Project>
